app/src/features/reports/VariableReport.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from "react";
+import { Grid } from "@material-ui/core";
+import TableChart from "../charts/TableChart";
+import styles from "./Report.module.scss";
+import WithDatasets from "../../utils/WithDatasets";
+import useDatasetStore from "../../utils/useDatasetStore";
+import variableProviders, { VariableId } from "../../utils/variableProviders";
+import { DropdownVarId } from "../../utils/MadLibs";
+import { Breakdowns } from "../../utils/Breakdowns";
+import VariableProvider from "../../utils/variables/VariableProvider";
+import { USA_FIPS } from "../../utils/Fips";
+import MapNavChart from "../charts/MapNavChart";
+import Alert from "@material-ui/lab/Alert";
+import Card from "@material-ui/core/Card";
+
+// TODO- investigate type check error to see if we can remove
+const VARIABLE_DISPLAY_NAMES: Record<
+  DropdownVarId,
+  Record<VariableId, string>
+> = {
+  // @ts-ignore
+  diabetes: {
+    // @ts-ignore
+    diabetes_count: "Diabetes Case Count",
+  },
+  // @ts-ignore
+  copd: {
+    // @ts-ignore
+    copd_count: "COPD Case Count",
+  },
+};
```

Comment:
If you need to override the typescript compiler, it's better to use a typecast which at least forces a specific override rather than generally silencing all compiler issues